modules/vba_documents/app/workers/vba_documents/upload_processor.rb

```diff
@@ -26,6 +26,7 @@ def perform(guid)
       upload = VBADocuments::UploadSubmission.find_by(guid: guid)
       tempfile, timestamp = download_raw_file(guid)
       begin
+        validate_parent(tempfile.path)
```

Comment:
I'm really confused by this sorry. It seems like the entire package should generally not be a valid pdf, or only interpreted as a valid pdf by a parser if it ignores the headers and that it's multiple pdfs concatenated together, which some pdf parsers would do since they generally try to display something.

Reply:
It was my assumption that the concat'd pdf's would still be a valid pdf. After pulling a valid one from the S3 it appears to clear the checks of pdfinfo. Judging by our other conversation, it looks like there may be more to it though.